apps/extension/src/hooks/last-block-synced.ts

```diff
@@ -22,7 +21,8 @@ export const useSyncProgress = () => {
   useQuery({
     queryKey: ['sync-blocks', grpcEndpoint],
     queryFn: async () => {
-      await internalSwClient.syncBlocks();
+      const syncing: boolean = await chrome.runtime.sendMessage('PENUMBRA_SYNC_BLOCKS');
```

Comment:
This feels like a worse dev experience / less type safe. We should encapsulate Std messages in one place somewhere. I expect we will need more Std messages as time goes by too.

Reply:
i agree we should have something convenient, this was a shortcut to demonstrate removing stdclient was imminently possible

i held stdclient removal back from routing-refactor because i knew this would come up. a simpler interface can be developed in this pr.